templates/activity_stream/tile_impressions/view.sql

```diff
@@ -0,0 +1,82 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.activity_stream.tile_impressions`
+AS
+WITH unioned AS (
+  SELECT
+    *
+  FROM
+    `moz-fx-data-shared-prod.activity_stream_stable.impression_stats_v1`
+  UNION ALL
+  SELECT
+    *
+  FROM
+    `moz-fx-data-shared-prod.activity_stream_stable.impression_stats_v1`
+  WHERE
+    DATE(submission_timestamp) = CURRENT_DATE()
```

Comment:
We could provide separate views for historical and live data, or we can union the two tables like this, which will likely be more convenient if it performs well.

Reply:
That would be awesome if performance is not a constraint!